packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -88,6 +90,12 @@ export type CallAdapterClientState = {
    * Azure communications Phone number to make PSTN calls with.
    */
   alternateCallerId?: string;
+  /* @conditional-compile-remove(rooms) */
+  /**
+   * Optional role of the local user that is assumed before a call is started. This role will determine permissions
```

Comment:
Please update this comment to very clearly indicate how `roleHint` is used. In particular, explain why this is named _hint_: That this is used only in the situation when role information is not available from the backend.